### ISSUE TITLE
Some (perhaps Mac-only) ideas

### ISSUE THREAD

**Issue by melyux** [openscopeproject/TrguiNG#17]
Beautiful app, already nicer looking than the mess that Transmission Remote GUI is.

Here are some issues/complaints I've found:
* Command + clicking torrents doesn't select them like it's supposed to do in Mac apps. Tried Ctrl + click since sometimes developers don't localize to the Mac, and it worked, but it also clashes with Mac's Ctrl + click = right click functionality.
* Speeds that are 0 should be hidden to avoid clutter (like Transmission Remote GUI does)
* Last active could be displayed as a "2h ago" instead of the bare datetime
* And of course, opening .torrent files with this app doesn't work

**Comment by qu1ck** [comment]
Thanks for feedback. First 2 are fixed and will be in next version. Last active is more useful as a date I think because usually you only care about the old ones to purge from the client so you sort by date and pick a cut off line. 

For the last one see #2. It's not supported by the framework I'm using right now, I may have to come up with some sort of hack.

**Comment by melyux** [comment]
Awesome, saw the fixes. For last date, is that so? I use it to understand how stale a download is. It's hard to parse the full date and then think of the current date and subtract in my head. Whereas "2d ago" would be much clearer. Maybe a way to click on it to switch between absolute and relative time?

**Comment by qu1ck** [comment]
But it doesn't indicate stale download, it indicates when was it last downloaded OR uploaded.

If you had the "2d ago" info, what do you typically use that for?

I could add another column that is just different representation of last active field and let users pick which column they want but it has to be justified by a good use case as there are already lots of columns.

**Comment by melyux** [comment]
Yep, I meant like how stale this torrent is. Has it downloaded or uploaded recently? 30d ago? If it's that stale, that's what makes me decide to remove it or not.

**Comment by qu1ck** [comment]
Right, so same as my use case, delete old ones. But you don't sort by the column, just pick out old ones and need an easier way to determine age at a glance?

**Comment by melyux** [comment]
Yeah I sort by age (aka datetime added) to keep a consistent ordering no matter what, so last active becomes much more important to quickly see at a glance since it's unpredictable by the order

**Comment by qu1ck** [comment]
I tried the relative time and found that it's just as easy to use with sorted column and easier to use with unsorted so I'll stick to it. Absolute date is still shown as tooltip on hover.

**Comment by melyux** [comment]
Build and tried latest. Everything's great
